templates/insights/insights_page.html

```diff
@@ -12,6 +12,14 @@
     {% for block in page.body %}
         {% include_block block %}
     {% endfor %}
+    {# Check if there are tags #}
+    {% if page.tags.count %}
+        <h1>Tags:</h1>
+        {% for tag in page.tags.all %}
+            {# Loop through all the existing tags #}
+            <a href="{{ self.get_parent.url }}?tag={{ tag.slug }}">{{ tag }}</a><br />
+        {% endfor %}
+    {% endif %}
```

Comment:
See https://national-archives.atlassian.net/browse/UN-140 for details on what needs to be output. I think this HTML can be removed from here.

Reply:
Thank you Andy , I picked this one from the online tutorial by Kalob, It was only for testing but it found its way here.